doc/queries.md

```diff
@@ -340,6 +341,209 @@ Result:
 }
 ```
 
+<a name="aggs-nested"></a>
+
+### 4. Nested Aggregation
+Guppy supports nested aggregations (sub-aggregations) for fields. Currently Guppy only supports two-level-sub-aggregations.
+There are two types of nested aggregations that is supported by Guppy: terms aggregation and missing aggregation, user can mix-and-match the using of both aggregations.
+
+#### 4.1. Terms Aggregation
+Terms aggregation returns histogram for a single `field` an array of fields. It means for each of the `key` of the single `field`, what is the distribution of each element from the array of fields. Results are wrapped by keywords `field` and also `key` and `count` for that `field`, example:
```

Comment:
```for a single `field` an array of fields``` i don't understand the sentence

Reply:
rephrased in https://github.com/uc-cdis/guppy/pull/43/commits/22746ef9dfaea91f37a87cd03f5bb459dd5109e2